sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -28,12 +27,12 @@ export interface SpanConfig {
  * @param tracingOptions - The options for the underlying http request.
  */
 export function createSpanFunction({ packagePrefix, namespace }: SpanConfig) {
-  return function<T extends OperationOptions>(
+  return function<T extends { tracingOptions?: OperationTracingOptions } | undefined>(
```

Comment:
do we need `| undefined`? It looks like it is redundant with `?`

Reply:
I think it's so someone can call the function with undefined itself, not that `tracingOptions` can be undefined. I am ambivalent